.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Question
+about: Who, what, when, where, and how?
+
+---
+
+Hey, stop right there!
+
+We use GitHub to track feature requests and bug reports. Please do not submit issues for questions about how to use JanusGraph, how to set JanusGraph up, best practices, or development related help.
+
+However, we do want to help! Head on over to our mailing lists and ask
+your questions there. In additional to awesome, knowledgeable community
+contributors, core JanusGraph developers check the mailing list.
+
+General questions regarding the Gremlin graph traversal language in general should be raised in the [Gremlin users group](https://groups.google.com/forum/#!forum/gremlin-users).
+
+Mailing list for troubleshooting, usage, setup, or best practices is https://groups.google.com/forum/#!forum/janusgraph-users.
+Mailing list for internal implementation details of JanusGraph, proposing a major new feature or subproject, or development releted help is https://groups.google.com/forum/#!forum/janusgraph-dev (do not post usage questions or bug reports in this thread)
```

Comment:
Please consider hiding the URL behind the list name, i.e., `[janusgraph-dev](https://groups.google.com/forum/#!forum/janusgraph-dev)`.